clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -756,8 +756,12 @@ runSYCLPostLinkTool(ArrayRef<StringRef> InputFiles, const ArgList &Args) {
     if (!ImageFileOrErr)
       return ImageFileOrErr.takeError();
 
+    // Arbitrary values are used for the testing of SYCL Offload Wrapping.
```

Comment:
`dry-run` != testing.

I don't think it's a good idea to add random properties in dry-run mode.